relay-server/src/utils/dynamic_sampling.rs

```diff
@@ -254,6 +260,112 @@ mod tests {
         envelope
     }
 
+    fn state_with_rule_and_condition(
+        sample_rate: Option<f64>,
+        rule_type: RuleType,
+        mode: SamplingMode,
+        condition: RuleCondition,
+    ) -> ProjectState {
+        let rules = match sample_rate {
+            Some(sample_rate) => vec![SamplingRule {
+                condition,
+                sample_rate,
+                ty: rule_type,
+                id: RuleId(1),
+                time_range: Default::default(),
+            }],
+            None => Vec::new(),
+        };
+
+        state_with_config(SamplingConfig {
+            rules,
+            mode,
+            next_id: None,
+        })
+    }
+
+    ///
+    ///
```

Comment:
Let's either remove these, or better yet, write a short description of what the test does.